tools/state-viewer/src/latest_witnesses.rs

```diff
@@ -1,12 +1,36 @@
+use std::path::{Path, PathBuf};
 use std::rc::Rc;
 
-use clap::Parser;
-use near_chain::ChainStore;
+use near_async::time::Clock;
+use near_chain::runtime::NightshadeRuntime;
+use near_chain::stateless_validation::processing_tracker::ProcessingDoneTracker;
+use near_chain::{Chain, ChainGenesis, ChainStore, DoomslugThresholdMode};
+use near_epoch_manager::shard_tracker::{ShardTracker, TrackedConfig};
+use near_epoch_manager::EpochManager;
+use near_primitives::stateless_validation::ChunkStateWitness;
 use near_primitives::types::EpochId;
 use near_store::Store;
 use nearcore::NearConfig;
+use nearcore::NightshadeRuntimeExt;
 
-#[derive(Parser)]
+#[derive(clap::Subcommand)]
+pub enum StateWitnessCmd {
+    /// Prints latest state witnesses saved to DB.
+    Latest(LatestWitnessesCmd),
```

Comment:
Not sure if there is any existing convention to call it that but to me it's very confusing. Should that be called something like Dump/Print/Get/Write/Show WitnessCmd?